jumpcloud-ADMU/Powershell/Private/UserIdentification/Test-UserFolderRedirect.ps1

```diff
@@ -58,11 +58,12 @@ function Test-UserFolderRedirect {
                 $folderRegKeyValue = (Get-Item -path $fullPath ).GetValue($folderRegKey , '', 'DoNotExpandEnvironmentNames')
                 $defaultRegFolder = "%USERPROFILE%\$userFolder"
```

Comment:
Do we need this line any longer? `$defaultRegFolder` no longer is used. IMO let's just turn this into a comment
 `# the default registry directory value is "%USERPROFILE%\$userFolder"`
You can leave that in the code comments but let's get rid of things we are no longer using

Reply:
Removed